### PR TITLE
Feature/resource oracle database cloud vm cluster

### PR DESCRIPTION
**Pull request by tulika-aakriti**
Part of: https://github.com/hashicorp/terraform-provider-google/issues/19701

**Release Note Template for Downstream PRs (will be copied)**

```release-note:new-resource
`google_oracle_database_cloud_vm_cluster`
```

### CODE REVIEW

File: mmv1/products/oracledatabase/CloudVmCluster.yaml [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+docs:
+id_format: 'projects/{{project}}/locations/{{location}}/cloudVmClusters/{{cloud_vm_cluster_id}}'
+base_url: 'projects/{{project}}/locations/{{location}}/cloudVmClusters'
+self_link: 'projects/{{project}}/locations/{{location}}/cloudVmClusters/{{cloud_vm_cluster_id}}'
+create_url: 'projects/{{project}}/locations/{{location}}/cloudVmClusters?cloudVmClusterId={{cloud_vm_cluster_id}}'
+immutable: true
+import_format:
+  - 'projects/{{project}}/locations/{{location}}/cloudVmClusters/{{cloud_vm_cluster_id}}'
+timeouts:
+  insert_minutes: 240
```
Comment (hao-nan-li) [inline mmv1/products/oracledatabase/CloudVmCluster.yaml:31]: Any reason to set the timeout minutes to 240 minutes and 120 minutes?

File: mmv1/products/oracledatabase/CloudVmCluster.yaml [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+      insert_minutes: 240
+      update_minutes: 120
+      delete_minutes: 120
+  result:
+    path: 'response'
+    resource_inside_response: true
+  error:
+    path: 'error'
+    message: 'message'
+custom_code:
```
Comment (hao-nan-li) [inline mmv1/products/oracledatabase/CloudVmCluster.yaml:52]: Is this line necessary here?

Comment (tulika-aakriti) [reply mmv1/products/oracledatabase/CloudVmCluster.yaml:52]: It's autogenarted I will remove and try.

**Comment by hao-nan-li** [comment]
Could you add the missing fields to your tests? Just to make sure every field is covered.

**Comment by hao-nan-li** [comment]
I think `version` is still needed for the tests

**Comment by tulika-aakriti** [comment]
> I think `version` is still needed for the tests

I couldn't find any similar example with version, if it's a blocker I can remove the field from yaml file itself as it's not a required field for our API.

**Review by hao-nan-li — APPROVED** [review]

**Pull request merged**